packages/csharp/PortingAssistant/PortingAssistant.Telemetry/Utils/LogUploadUtils.cs

```diff
@@ -1,254 +1,73 @@
-using Amazon.Runtime;
-using Amazon.Runtime.CredentialManagement;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 using PortingAssistantExtensionTelemetry.Model;
 using System;
-using System.Collections;
-using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Net;
-using System.Net.Http;
 using System.Net.NetworkInformation;
-using System.Text;
-using System.Threading.Tasks;
 using PortingAssistant.Client.Telemetry;
-using Amazon;
+using Serilog;
 
 namespace PortingAssistant.Telemetry.Utils
 {
     public static class LogUploadUtils
     {
-        private static Boolean IsFileLocked(FileInfo file)
+        private static string GetLogName(string file)
         {
-            FileStream stream = null;
+            var fName = Path.GetFileNameWithoutExtension(file);
+            var logName = "";
 
-            try
+            string typeOfLog = (fName.Split('-').Length > 1) ? fName.Split('-')[1] : String.Empty;
+
+            if (typeOfLog == "telemetry")
```

Comment:
use this for string compare FirstString.Equals(SecondString, StringComparison.OrdinalIgnoreCase))

Reply:
or [String.Compare(first, second, StringComparison.OrdinalIgnoreCase)](https://docs.microsoft.com/en-us/dotnet/api/system.string.compare?view=net-6.0#system-string-compare(system-string-system-string-system-stringcomparison)) :)

Reply:
thanks, have made the change below